packages/test/test-service-load/src/runner.ts

```diff
@@ -386,6 +391,20 @@ async function setupOpsMetrics(container: IContainer, logger: ITelemetryLogger,
         }
     });
 
+    let submittedSignals = 0;
+    testRuntime.on("signal", (message: IInboundSignalMessage, local: boolean) => {
+        if (message.type === "generic-signal" && local === true) {
+            submittedSignals += 1;
+        }
+    });
+
+    let receivedSignals = 0;
+    testRuntime.on("signal", (message: IInboundSignalMessage, local: boolean) => {
+        if (message.type === "generic-signal" && local === false) {
```

Comment:
why don't we want to count the receiving of our own singals?

Reply:
Local signals are handled locally, and are not sent to PUSH so we don't count them.